doc/user/content/sql/functions/now_and_mz_logical_timestamp.md

```diff
@@ -19,64 +19,24 @@ In Materialize, `now()` doesn't represent the system time, as it does in most sy
 
 ## Temporal filters
 
-You can use `mz_logical_timestamp()` to define temporal filters for materialized view, or computations for fixed windows of time. For more information, see [Temporal Filters: Enabling Windowed Queries in Materialize](https://materialize.com/temporal-filters/).
+You can use `mz_logical_timestamp()` to define temporal filters for materialized view, or computations for fixed windows of time.
 
-## Example
+For more information, see [Temporal Filters: Enabling Windowed Queries in Materialize](https://materialize.com/temporal-filters/).
 
-### Temporal filter using mz_logical_timestamp()
+## Restrictions
 
-For this example, you'll need to create a sample data source and create a materialized view from it for later reference.
+You can only use `mz_logical_timestamp()` to establish a temporal filter in one of the following types of clauses:
 
-```sql
---Create table
```

Comment:
nit: the 2 sql comments seem a bit unnecessary

Reply:
I think it's helpful to break up the big chunks of code.